discovery/kubernetes/pod.go

```diff
@@ -222,6 +224,15 @@ func podLabels(pod *apiv1.Pod) model.LabelSet {
 		podUID:           lv(string(pod.UID)),
 	}
 
+	for _, podIP := range pod.Status.PodIPs {
```

Comment:
Would it be worth commenting that this is expected to contain at most two addresses?